examples/features/standard/interceptor-client/pom.xml

```diff
@@ -36,11 +36,6 @@ under the License.
    </properties>
 
    <dependencies>
-      <dependency>
-          <groupId>org.apache.activemq.examples.broker</groupId>
-          <artifactId>interceptor</artifactId>
-          <version>${project.version}</version>
-      </dependency>
```

Comment:
Since this was removed without adding anything...does it just mean this example doesnt _actually_ depend on the other one, but its just mistakenly said it did all this time?

Reply:
Yes, I think so